loadtest/user/userentity/actions.go

```diff
@@ -325,4 +325,9 @@ func (ue *UserEntity) GetTeamsUnread(teamIdToExclude string) ([]*model.TeamUnrea
 	}
 
 	return unread, nil
+}
+
+func (ue *UserEntity) GetFileInfosForPost(postId string) ([]*model.FileInfo, error) {
+	infos, response := ue.client.GetFileInfosForPost(postId, "")
+	return infos, response.Error
```

Comment:
Same as the [previous](https://github.com/mattermost/mattermost-load-test-ng/pull/34) PR, check for the error and rename response variable.